### DISCUSSION TITLE
Bewlybewly for youtube

### DISCUSSION THREAD

**Discussion by yepengfan** [BewlyBewly/BewlyBewly#239] [Ideas]
Hi @hakadao 

Thank you for creating such a fantastic browser extension. I would like to know if there is a similar extension available for YouTube, or if you have any plans to support YouTube in the future?

Thanks!

**Comment by hakadao** [comment]
Thank you for your appreciation!
BewlyBewly is mostly inspired by YouTube. The current style of YouTube is modernized and has an avant-garde design. I think the current style of YouTube is enough.
